development/tools/jobs/kyma/load_test_test.go

```diff
@@ -0,0 +1,65 @@
+package kyma_test
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadTestJobsPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/tools/load-test/load-test.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "kyma-tools-load-test"
+	assert.Equal(t, expName, actualPresubmit.Name)
+	assert.Equal(t, []string{"master"}, actualPresubmit.Branches)
+	assert.Equal(t, 10, actualPresubmit.MaxConcurrency)
+	assert.True(t, actualPresubmit.SkipReport)
+	assert.True(t, actualPresubmit.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPresubmit.PathAlias)
+	tester.AssertThatHasExtraRefTestInfra(t, actualPresubmit.JobBase.UtilityConfig)
+	tester.AssertThatHasPresets(t, actualPresubmit.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepo, tester.PresetGcrPush, tester.PresetBuildPr)
+	tester.AssertThatJobRunIfChanged(t, actualPresubmit, "tools/load-test/fix")
+	assert.Equal(t, "^tools/load-test/", actualPresubmit.RunIfChanged)
+	assert.Equal(t, tester.ImageGolangBuildpackLatest, actualPresubmit.Spec.Containers[0].Image)
+	tester.AssertThatHasCommand(t, actualPresubmit.Spec.Containers[0].Command)
+	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"}, actualPresubmit.Spec.Containers[0].Command)
+	assert.Equal(t, []string{"/home/prow/go/src/github.com/kyma-project/kyma/tools/load-test"}, actualPresubmit.Spec.Containers[0].Args)
+}
+
+func TestLoadTestJobPostsubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/tools/load-test/load-test.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Postsubmits, 1)
+	kymaPost, ex := jobConfig.Postsubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPost, 1)
+
+	actualPost := kymaPost[0]
+	expName := "kyma-tools-load-test"
+	assert.Equal(t, expName, actualPost.Name)
+	assert.Equal(t, []string{"master"}, actualPost.Branches)
+
+	assert.Equal(t, 10, actualPost.MaxConcurrency)
+	assert.True(t, actualPost.Decorate)
+	assert.Equal(t, "github.com/kyma-project/kyma", actualPost.PathAlias)
+	tester.AssertThatHasExtraRefTestInfra(t, actualPost.JobBase.UtilityConfig)
+	tester.AssertThatHasPresets(t, actualPost.JobBase, tester.PresetDindEnabled, tester.PresetDockerPushRepo, tester.PresetGcrPush, tester.PresetBuildMaster)
+	assert.Equal(t, "^tools/load-test/", actualPost.RunIfChanged)
+	assert.Equal(t, tester.ImageGolangBuildpackLatest, actualPost.Spec.Containers[0].Image)
+	tester.AssertThatHasCommand(t, actualPost.Spec.Containers[0].Command)
```

Comment:
Could you also add an assertion to check if the command is correct like in presubmit?